bot/cogs/commands/useful.py

```diff
@@ -664,7 +664,7 @@ async def remind_me(self, ctx: Ctx, duration_str: str, *, reminder: str):
             reminder[:499].replace("@everyone", "@\uFEFFeveryone").replace("@here", "@\uFEFFhere"),
             at.datetime,
         )
-        unix_timestamp = format_dt(at.datetime, style="r")
+        unix_timestamp = format_dt(at.datetime, style="R")
```

Comment:
Since this line isn't very long, might as well just remove the unnecessary variable and move this down to where it's actually used.